src/api/spec/rails_helper.rb

```diff
@@ -26,6 +26,10 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect  # just use the new syntax
```

Comment:
@ChrisBr this is the same as in your PR, but you did it in the spec_helper.rb file... So if yours get merged before mine I will drop this change